main.go

```diff
@@ -72,6 +74,9 @@ func main() {
 		}
 
 		for _, node := range nodeList.Items {
+			if _, ok := node.Labels[masterNodeLabel]; ok {
+				continue
+			}
```

Comment:
instead of ignoring the iteration, I guess we can fetch all the worker nodes like below

`workerNodeList, err := clientset.CoreV1().Nodes().List(context.TODO(), meta_v1.ListOptions{LabelSelector: "!node-role.kubernetes.io/master"})`

Reply:
good catch :)
fixed